apps/mark/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -405,14 +399,14 @@ export function PollWorkerScreen({
                 {getPollTransitionsFromState(pollsState).map(
                   (pollsTransition, index) => {
                     return (
-                      <React.Fragment
-                        key={`${pollsTransition}-directly-button`}
-                      >
-                        {index > 0 && ' or '}
-                        <UpdatePollsDirectlyButton
-                          pollsTransition={pollsTransition}
-                          updatePollsState={updatePollsState}
-                        />
+                      <React.Fragment key={`${pollsTransition}-button`}>
+                        <P>
+                          <UpdatePollsButton
+                            pollsTransition={pollsTransition}
+                            updatePollsState={updatePollsState}
+                            isPrimaryButton={index === 0}
+                          />
+                        </P>
                       </React.Fragment>
```

Comment:
I believe this `React.Fragment` is no longer needed. We can get rid of it and add the key to the `P` element